great_expectations/execution_engine/util.py

```diff
@@ -19,6 +19,48 @@
     WithinGroup = None
 
 
+# noinspection PyPep8Naming
+def get_or_create_spark_session(
+    name: str = "default_great_expectations_spark_dataframe_execution_engine",
```

Comment:
???

Reply:
@jcampbell This is a default when the user DOES NOT specify spark_config -- and hence we have to come up with a default `spark.app.name` value.